CHANGELOG.md

```diff
@@ -34,6 +34,22 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
+## [Unreleased]
+
+### State Machine Breaking
+[\#1158](https://github.com/Kava-Labs/kava/pull/1158) Split existing auction `bid_duration` parameter into `forward_bid_duration` and `reverse_bid_duration`
+
+## [v0.16.1] 
```

Comment:
Release tag links for some versions are missing, could be done separately if we want to restructure the links to reference-style